### PR TITLE
MBS-9909: Change reports to use React (9 - URLs)

### PR DESCRIPTION
**Pull request by reosarevok**
To be reviewed and merged after https://github.com/metabrainz/musicbrainz-server/pull/918

### CODE REVIEW

File: lib/MusicBrainz/Server/Report/DeprecatedRelationshipUrls.pm [hunk 1]
```diff
@@ -1,4 +1,4 @@
-package MusicBrainz::Server::Report::DeprecatedRelationshipURLs;
+package MusicBrainz::Server::Report::DeprecatedRelationshipUrls;
```
[the remark this answers is not in the document]
Comment (reosarevok) [reply lib/MusicBrainz/Server/Report/DeprecatedRelationshipUrls.pm]: It's consistent with what we already did in the past with Iswc etc. Right now the Perl and JS names are supposed to be equal. I could change that, but is there a need?

[the remark this answers is not in the document]
Comment (yvanzo) [reply lib/MusicBrainz/Server/Report/DeprecatedRelationshipUrls.pm]: Probably yes cause it is also changing report’s URL. Sorry to have missed that with ISWC, ISRC and previous URL occurrences. VA has not been changed though.

[the remark this answers is not in the document]
Comment (reosarevok) [reply lib/MusicBrainz/Server/Report/DeprecatedRelationshipUrls.pm]: Fixed these

**Comment by reosarevok** [comment]
For the new eslint indent rules: I'm changing this for everything in https://github.com/metabrainz/musicbrainz-server/pull/934/commits/d84e6ce1f0763d85aabf9e02cecb4e502b1a0c68 with one --fix

**Comment by mwiencek** [comment]
> For the new eslint indent rules: I'm changing this for everything in [d84e6ce](https://github.com/metabrainz/musicbrainz-server/commit/d84e6ce1f0763d85aabf9e02cecb4e502b1a0c68) with one --fix

LGTM but it'll create a massive number of conflicts with #801, so it'd be nice to hold off for a bit. :)

**Comment by reosarevok** [comment]
I'd really hope we merge that before the last of these, anyway.

File: lib/MusicBrainz/Server/Report/DeprecatedRelationshipUrls.pm (hunk 1 above)
[the remark this answers is not in the document]
Comment (yvanzo) [reply lib/MusicBrainz/Server/Report/DeprecatedRelationshipUrls.pm]: Thank you!

**Pull request merged**